includes/class-payment-information.php

```diff
@@ -223,6 +223,12 @@ public static function from_payment_request(
 		$token            = self::get_token_from_request( $request );
 		$cvc_confirmation = self::get_cvc_confirmation_from_request( $request );
 
+		if ( isset( $request['is_woopay'] ) && $request['is_woopay'] ) {
+			$order->add_meta_data( 'is_woopay', true );
+			$order->add_meta_data( 'last4', $request['last4'] );
```

Comment:
I mentioned this in the other PR reviews, but I'm seeing `last4` appear twice in the order meta for WooPay orders, when I use both PRs' changes on both the WooPay site and the merchant site. Noting that here too since this line may be related. We may also want to check to make sure that `last4` is set before using it to update order metadata.

Reply:
I found another solution here. So we don't need to save the last4 in WooPay and then send it to WC Payments. Now we only send the is_woopay flag.

Fixed with 8b9748e6f2b4fee6c30728fdc77ff50f53f0b62e